src/engine/instrument.h

```diff
@@ -48,6 +48,7 @@ enum DivInstrumentType {
   DIV_INS_BEEPER=21,
   DIV_INS_SWAN=22,
   DIV_INS_MIKEY=23,
+  DIV_INS_X1_010=24,
```

Comment:
Please change to 25 to allow this: https://github.com/tildearrow/furnace/pull/253/files#diff-6fc56b950a9e5d1491038859f2efb3c5ef6203e098d813d9b3c1c43bb8ff9b24R51